src/forgot-password/ForgotPasswordPage.jsx

```diff
@@ -109,8 +110,15 @@ const ForgotPasswordPage = (props) => {
                   onClick={handleSubmit}
                   onMouseDown={(e) => e.preventDefault()}
                 />
-                <Hyperlink id="forgot-password" className="btn btn-link font-weight-500 text-body" destination={getConfig().LOGIN_ISSUE_SUPPORT_LINK}>
-                  {intl.formatMessage(messages['need.help.sign.in.text'])}
+                <Hyperlink
+                  id="forgot-password"
+                  className="btn btn-link font-weight-500 text-body"
```

Comment:
Remove `btn btn-link` and add `ml-4` fixes the underline issue. You can remove the custom styles with this.

Reply:
sounds good. Will push the change

Reply:
done